dataactvalidator/config/sqlrules/a34_appropriations.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+    approp.row_number,
+    approp.budget_authority_unobligat_fyb,
+    sf.amount as sf_133_amount
```

Comment:
Agreed that the rule text is enough context for people to track this down, so we're all good here.

In general, though, I don't think we need to prioritize consistency of past code over making improvements. I'd hate for the notion that we have to retrofit every enhancement to prevent us from doing incremental improvements!